frontend/src/metabase/lib/formatting/dateTimeUtils.ts

```diff
@@ -0,0 +1,34 @@
+import type { DatetimeUnit } from "metabase-types/types/Query";
```

Comment:
Types from `metabase-types/types/**` considered to be legacy onces and should be avoided because they are not exactly correct